src/test/java/org/gridsuite/actions/server/ContingencyListControllerTest.java

```diff
@@ -132,6 +136,8 @@ private void assertQueuesEmptyThenClear(List<String> destinations, OutputDestina
 
     @Before
     public void setUp() {
+        objectMapper.registerModule(new JavaTimeModule());
+        objectMapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
```

Comment:
Dunno why CheckStyle didn't flag this, but move variable [L394](https://github.com/gridsuite/actions-server/blob/main/src/test/java/org/gridsuite/actions/server/ContingencyListControllerTest.java#L394) to lines ~100, and use `@Autowired` instead of new(). You should have modules registered for you by Spring.